src/Tickets/Commerce/Order_Modifiers/Modifier_Admin_Handler.php

```diff
@@ -279,23 +280,26 @@ protected function render_edit_view( Modifier_Manager $manager, array $context )
 		// Get modifier ID from the context.
 		$modifier_id = (int) $context['modifier_id'];
 
-		// Merge the modifier data into the context to be passed to the form rendering logic.
-		// If a valid modifier ID is provided, fetch the data to populate the form.
-		if ( $modifier_id > 0 ) {
-			$modifier_data = $this->get_modifier_data_by_id( $modifier_id );
+		if ( 0 > $modifier_id ) {
```

Comment:
Should this be `>=`?

Reply:
No for new modifiers being added

Reply:
Check also tests that are not failing

Reply:
The previous condition was `$modifier_id > 0`. The inverted condition should be `0 >= $modifier_id`, since `0` is also invalid.

Reply:
I explained that 0 is for new modifiers. Please test it out in your local to understand it